build/Microsoft.IdentityModel.Clients.ActiveDirectory.nuspec

```diff
@@ -18,8 +18,8 @@
     <dependencies>
 		<group targetFramework="netstandard1.4">
 			<dependency id="NETStandard.Library" version="1.6" />
-			<dependency id="System.Runtime.Serialization.Json" version="4.0.2" />
-			<dependency id="System.Runtime.Serialization.Primitives" version="4.1.1" />
+			<dependency id="System.Runtime.Serialization.Json" version="4.3.0" />
```

Comment:
Miscellaneous: the hard-coded nuspec file was out of date: the ADAL.Net project actually references these version.